### PR TITLE
Deep links

### LINKED ISSUE #57: Ensure all screens are deep-linkable / url safe
My CLC (and others) should be able to be linked to via push notifications or similar. The space in the screen key does not allow for this

clc://clc/app-link/Schedule

### PR DESCRIPTION
**Pull request by gburgett**
* Creates a report to generate deep links to all content in the app by querying Contentful

See attached for current report
[deep-links.csv](https://github.com/watermarkchurch/clc-app-v2/files/8365988/deep-links.csv)

fixes #57

### CODE REVIEW

**Comment by jpowell** [comment]
image

Is this a typo / hard tab on csv or a bug?

File: reports/src/app-links.ts [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    const didEnd = onceAsync(
+      writerStream.pipe(outputStream),
+      'finish'
+    )
+    const alreadyWrote = new Set<string>()
+
+    const Tabs: AppLinkRow[] = [
+      { type: 'Tab', title: 'Home', link: 'clc://clc/app-link/Home'},
+      { type: 'Tab', title: 'Schedule', link:  'clc://clc/app-link/Schedule' },
+      { type: 'Tab', title: 'My CLC', link:  '	clc://clc/app-link/My-CLC' },
```
Comment (jpowell) [inline reports/src/app-links.ts:53]: image

**Review by jpowell — CHANGES_REQUESTED** [review]
Looks great - we'll also want announcements and anything the connect tab can link to

**Comment by gburgett** [comment]
Yep all that is queried through the Conference's direct links.  So it's captured here.

Fixed the typo.

**Pull request merged**